fbchat/client.py

```diff
@@ -157,11 +199,14 @@ def _postLogin(self):
         self.payloadDefault = {}
         self.client_id = hex(int(random()*2147483648))[2:]
         self.start_time = now()
-        self.uid = str(self._session.cookies['c_user'])
+        self.uid = self._session.cookies.get_dict().get('c_user')
```

Comment:
This would probably also fix #198, since `requests.session.cookies` works kinda wierd